client/components/guests.jsx

```diff
@@ -1,21 +1,100 @@
-import React, {Component} from 'react';
-import styles from './styles.js';
+import React, { Component } from 'react';
+import './css/app.css';
+import DropDown from './guests/dropdown.jsx';
 
 class Guests extends Component {
-  constructor(props){
+  constructor(props) {
     super(props);
+
+    this.state = {
+      toggled: false,
+      adults: 1,
+      children: 0,
+      infants: 0
+    }
+
+    this.showDropDown = this.showDropDown.bind(this);
+    this.increaseAdults = this.increaseAdults.bind(this);
+    this.decreaseAdults = this.decreaseAdults.bind(this);
+    this.increaseChildren = this.increaseChildren.bind(this);
+    this.decreaseChildren = this.decreaseChildren.bind(this);
+    this.increaseInfants = this.increaseInfants.bind(this);
+    this.decreaseInfants = this.decreaseInfants.bind(this);
+    this.getTotal = this.getTotal.bind(this);
+  }
+
+  showDropDown() {
+    this.setState({
+      toggled: !this.state.toggled
+    })
+  }
+
+  increaseAdults() {
+    this.setState({
+      adults: this.state.adults+1
+    })
+  }
+
+  decreaseAdults() {
+    if(this.state.adults > 1){
+      this.setState({
+        adults: this.state.adults-1
```

Comment:
You should use the callback form of `this.setState` if you are updating the state based on its current/previous version. https://reactjs.org/docs/react-component.html#setstate